src/foam/core/Static.js

```diff
@@ -0,0 +1,61 @@
+/**

```

Comment:
Year is 2017 and we're not Google, so the header should be:

/**
 * @license
 * Copyright 2017 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */

Reply:
Use shorter header shown above.